### ISSUE TITLE
allow to use system expat

### ISSUE THREAD

**Issue by VolkerEnderlein** [coin3d/coin#142]
**[Original report](https://bitbucket.org/Coin3D/coin/issue/142) by Lorenz L. (Bitbucket: [lorenz_l](https://bitbucket.org/lorenz_l), ).**

Attachments: [hg.diff](https://github.com/coin3d/attachments/blob/master/Coin3D_coin/issue142/hg.diff)

----------------------------------------

I can't create a PR, because cloning my fork doesn't work.
abort: unknown revision 'd12bf6cbb77c33f3023eae9dc8ae08c782eb862f'!

Therefor I ask for inclusion of the appended patch-file.
Basically it allows to use a system expat library.
